File: core/src/main/java/com/uiptv/service/remotesync/DefaultRemoteSyncUiBridge.java
package com.uiptv.service.remotesync;

import com.uiptv.util.AppLog;
import java.util.function.Consumer;

public class DefaultRemoteSyncUiBridge implements RemoteSyncApprovalPrompt, RemoteSyncNotifier {
    @Override
    public void requestApproval(RemoteSyncApprovalRequest request, Consumer<Boolean> decisionConsumer) {
        decisionConsumer.accept(false);
    }

    @Override
    public void showInfo(String message) {
        AppLog.addInfoLog(DefaultRemoteSyncUiBridge.class, message);
    }

    @Override
    public void showError(String message) {
        AppLog.addErrorLog(DefaultRemoteSyncUiBridge.class, message);
    }
}

File: server/src/main/kotlin/com/uiptv/service/remotesync/DefaultRemoteSyncUiBridge.kt
package com.uiptv.service.remotesync

import com.uiptv.util.AppLog
import java.util.function.Consumer

class DefaultRemoteSyncUiBridge : RemoteSyncApprovalPrompt, RemoteSyncNotifier {
    override fun requestApproval(request: RemoteSyncApprovalRequest, decisionConsumer: Consumer<Boolean>) {
        decisionConsumer.accept(false)
    }

    override fun showInfo(message: String) {
        AppLog.addInfoLog(DefaultRemoteSyncUiBridge::class.java, message)
    }

    override fun showError(message: String) {
        AppLog.addErrorLog(DefaultRemoteSyncUiBridge::class.java, message)
    }
}
